ci/jenkins/jobs/macros.yaml

```diff
@@ -231,6 +231,8 @@
           #!/bin/bash
           set -ex
           DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+          # Delete all kind clusters present from more than 135 mins
+          ./ci/kind/kind-setup.sh destroy "" "135m ago"
```

Comment:
Can we use "--until" instead of ""?

Reply:
actually here I am trying to pass cluster name empty string because we want delete multiple cluster not a single cluster and if we use  "--until" instead of "" , something like this (./ci/kind/kind-setup.sh destroy  "--until" "135m ago") then cluster name will be "--until" in that case.
but I think we can use  "--until" because this condition will take care of that :
```
  destroy)
      CLUSTER_NAME="$2"
      if [[ "$3" != "" ]]; then
         UNTIL=true
         TIME="$3"
      fi  
      destroy
      exit 0
```
 could you please correct me?

Reply:
+1 to @XinShuYang's comment. It should be similar to `kubectl delete` which can take positional parameters or optional parameters. For example `kubectl delete pod foo` deletes a single pod, and `kubectl delete pod --all` deletes all pods. 
But I understand it may be harder for bash to differentiate positional parameters and optional parameters. It may be good to add another subcommand like `prune` to clean up multiple clusters if there is no simpler solution.
cc @antoninbas who knows this better.

However, we need `"--until"` which is more for readability. Otherwise the usage is confusing.

Reply:
Hi @tnqn , @XinShuYang if we don't  want to add subcommand then I think one way to cover both cases ( delete single, multiple clusters) is my current implementation.
to delete single cluster:
`ci/kind/kind-setup.sh destroy ClusterName`
to delete multiple cluster:
`./ci/kind/kind-setup.sh destroy --until "135m ago"`

but if we want to add subcommand then I will  do that.

Reply:
I also think that it's not too difficult to support `destroy <name>` and `destroy --all`. It's not such a big change to the parsing logic. We just need to store positional arguments and unknown options, and process them at the end:
- starts with `-`: unknown option, error
- more than one positional argument: error
- etc.

Reply:
@tnqn  @XinShuYang  @antoninbas Thanks for your suggestions, I have updated the code please take a look. thanks!